Added manifest and service worker.

garajo opened the pull request:
I've added only the most basic implementation. But you can add it now to your desktop :)

Icon images should be replaced. Also I might have merged index.html wrong; I left a note.

Would resolve #3 and #6 

![screenshot from 2018-09-16 08-46-55](https://user-images.githubusercontent.com/2825366/45598296-7be68680-b98e-11e8-81e1-f3b53cfb77d7.png)

garajo-git committed:
Added manifest and service worker.

garajo-git committed:
Added manifest and service worker.

garajo commented on public/index.html:
```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>ID Simulator</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.min.css">
```
@anthkris Did I accidentally put this back in?

anthkris replied:
@garajo Yup. that shouldn't be necessary any more.

garajo replied:
ok, removed

garajo-git committed:
Removed uneeded link.

garajo-git committed:
Removed uneeded link.

garajo-git committed:
Removed bulma sass dependency.

garajo-git committed:
Removed bulma sass dependency.

ghost commented:
fyi, I ran lighthouse audit for PWA. Resolving 'start_url' and 'theme-colors' need to be defined.

![screenshot from 2018-09-16 17-15-18](https://user-images.githubusercontent.com/2825366/45602485-20d78280-b9d4-11e8-8335-fe9b982fc883.png)

garajo-git committed:
Put back bulma dependency.

garajo-git committed:
Put back bulma dependency.

anthkris committed:
refactor: Pull out several reusable code bits, begin redesigning game play

anthkris committed:
feat: Update gameplay logic

Refactor JS into separate files
Update gameplay logic such that:
- Each opportunity has some threshold object which may be empty or may have prerequisites (e.g. reputation must be >= 20)
- The logic will check for prerequisites (as well as status and turns to wait) but otherwise serve up one of any eligible opportunities (No turn will ever be empty)
Begin creating logic for previewing consequences

garajo-git committed:
Corrected dev port on readme.

garajo-git committed:
Added theme color meta.

garajo-git committed:
Merge branch 'webpackd' of github.com:garajo/id-simulator into webpackd

garajo-git committed:
Removed extra implementation.

garajo-git committed:
Resolved start url.

ghost commented:
To show the "Install" button to add the add to your desktop...

![screenshot from 2018-09-22 22-03-37](https://user-images.githubusercontent.com/2825366/45924538-9ac9a900-beb7-11e8-90fd-c5968bb5d6f5.png)

need to either play around the app until the browser think you want to install it or by explicitly clicking "Add to homescrenn" (in Chrome)...

![screenshot from 2018-09-22 22-03-44](https://user-images.githubusercontent.com/2825366/45924542-a321e400-beb7-11e8-9da3-1af4b83bb10b.png)

ghost commented:
@anthkris I've completed the major parts for the PWA check. You can merge this PR now :)

![screenshot from 2018-09-22 22-29-39](https://user-images.githubusercontent.com/2825366/45924563-39eea080-beb8-11e8-9bc1-74210da97fe9.png)
